docs/how-to/enforce-permissions/data-filtering.mdx

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_position: 4
+title: Data Filtering
+---
+
+Data filtering in permissions represents is a different kind of approach when enforcing permissions.
```

Comment:
Implementing data filtering within access control represents a different approach to managing permissions.

Reply:
fixed